pkg/storaged/client.js

```diff
@@ -391,9 +393,11 @@
                 $(client.storaged_client).on('notify', function () {
                     update_indices();
                     $(client).triggerHandler('changed');
+                    client.dispatchEvent("changed");
```

Comment:
I assume you did this so that there's no dependency on jQuery in the new files. Do you plan on moving everything over to `addEventListener`? It's a bit unwieldy to have both in here.

Reply:
Yes, but good news: jQuery will see events triggered by `disptachEvent`, so we can remove the call to `triggerHandler`.